javascript-carousel/main.js

```diff
@@ -0,0 +1,128 @@
+    dotThree.setAttribute('class', 'dot three fa-regular fa-circle');
+    dotTwo.setAttribute('class', 'dot two fa-solid fa-circle');
+  } else if (dotTwo.className === 'dot two fa-solid fa-circle') {
+    img.setAttribute('src', 'images/001.png');
+    dotTwo.setAttribute('class', 'dot two fa-regular fa-circle');
+    dotOne.setAttribute('class', 'dot one fa-solid fa-circle');
+  }
+});
+
+chevRight.addEventListener('click', function () {
+  if (dotOne.className === 'dot one fa-solid fa-circle') {
+    img.setAttribute('src', 'images/004.png');
+    dotTwo.setAttribute('class', 'dot two fa-solid fa-circle');
+    dotOne.setAttribute('class', 'dot one fa-regular fa-circle');
+  } else if (dotTwo.className === 'dot two fa-solid fa-circle') {
+    img.setAttribute('src', 'images/007.png');
+    dotTwo.setAttribute('class', 'dot two fa-regular fa-circle');
+    dotThree.setAttribute('class', 'dot three fa-solid fa-circle');
+  } else if (dotThree.className === 'dot three fa-solid fa-circle') {
+    img.setAttribute('src', 'images/025.png');
+    dotThree.setAttribute('class', 'dot three fa-regular fa-circle');
+    dotFour.setAttribute('class', 'dot four fa-solid fa-circle');
+  } else if (dotFour.className === 'dot four fa-solid fa-circle') {
+    img.setAttribute('src', 'images/039.png');
+    dotFour.setAttribute('class', 'dot four fa-regular fa-circle');
+    dotFive.setAttribute('class', 'dot five fa-solid fa-circle');
+  } else if (dotFive.className === 'dot five fa-solid fa-circle') {
+    img.setAttribute('src', 'images/001.png');
+    dotFive.setAttribute('class', 'dot five fa-regular fa-circle');
+    dotOne.setAttribute('class', 'dot one fa-solid fa-circle');
+  }
+});
+
+function toTheRight() {
+  if (dotOne.className === 'dot one fa-solid fa-circle') {
+    img.setAttribute('src', 'images/004.png');
+    dotTwo.setAttribute('class', 'dot two fa-solid fa-circle');
+    dotOne.setAttribute('class', 'dot one fa-regular fa-circle');
+  } else if (dotTwo.className === 'dot two fa-solid fa-circle') {
+    img.setAttribute('src', 'images/007.png');
+    dotTwo.setAttribute('class', 'dot two fa-regular fa-circle');
+    dotThree.setAttribute('class', 'dot three fa-solid fa-circle');
+  } else if (dotThree.className === 'dot three fa-solid fa-circle') {
+    img.setAttribute('src', 'images/025.png');
+    dotThree.setAttribute('class', 'dot three fa-regular fa-circle');
+    dotFour.setAttribute('class', 'dot four fa-solid fa-circle');
+  } else if (dotFour.className === 'dot four fa-solid fa-circle') {
+    img.setAttribute('src', 'images/039.png');
+    dotFour.setAttribute('class', 'dot four fa-regular fa-circle');
+    dotFive.setAttribute('class', 'dot five fa-solid fa-circle');
+  } else if (dotFive.className === 'dot five fa-solid fa-circle') {
+    img.setAttribute('src', 'images/001.png');
+    dotFive.setAttribute('class', 'dot five fa-regular fa-circle');
+    dotOne.setAttribute('class', 'dot one fa-solid fa-circle');
+  }
+}
```

Comment:
This is a _lot_ of code repetition. Please attempt to combine some of this in ways that let you re-use code between slides.